src/state/apis/zapper/validators.ts

```diff
@@ -182,6 +182,7 @@ type ZapperTokenBase = {
 // @ts-ignore zod/myzod is drunk https://github.com/colinhacks/zod/issues/577
 const ZapperTokenBaseSchema: Type<ZapperTokenBase> = z.intersection(
   z.object({
+    uuid: z.string().optional(),
```

Comment:
Not a mistake - token `uuid` may be undefined for some, but is required in the root assets